contributors/devel/sig-node/triage.md

```diff
@@ -0,0 +1,153 @@
+[trivial edits policy]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#trivial-edits
+
+### needs-priority
+
+You can take a quick look at what the PR is addressing and then apply a
+priority label.
+
+Anyone can add this label.
+
+- **priority/critical-urgent:** Urgent bug fix, required ASAP. If not
+  addressed, will block a release. These issues should always be discussed in
+  the `#sig-node` channel on Slack.
+- **priority/important-soon:** Needs to be completed this release. Important
+  bug fixes + KEPs targeted for the current milestone.
+- **priority/important-longterm:** Has an attached issue/KEP, but unclear what
+  the specific priority is.
+- **priority/backlog:** Non-urgent changes such as typo fixes, linters, small
+  comment fixes, etc.
+
+### needs-triage
+
+There are two more things to check before accepting a PR for triage.
+
+The first is whether the SIG is correct. If the PR does not appear to touch SIG
+Node code or require a SIG Node approver, you should remove the SIG Node label,
+and add other SIG labels as appropriate.
+
+The second is verifying the kind of PR. Most will be bug fixes, cleanups, or
+documentation. Feature PRs should generally have an attached KEP. API changes
+and deprecations require special review; those labels may be mistakenly
+applied, so check over the PR to make sure they're accurate.
+
+Once you've quickly looked over a PR, applied the appropriate labels, and it
+looks ready to proceed to review (i.e. it doesn't have any labels that would
+mean it's waiting on more work from the author), you can mark the PR as triaged
+with `/triage accepted`.
+
+Only [Kubernetes org members] can add this label.
+
+## Waiting on Author
+
+This column means that the PR is waiting on some action from the author. A
+reviewer may have requested changes, or a PR may have one of the following
+do-not-merge labels:
+
+- **do-not-merge/hold:** usually set by a reviewer
+- **do-not-merge/work-in-progress:** usually set by an author
+- **do-not-merge/release-note-needed:** needs a release note to be added by the
+  author, [Kubernetes org members] can override with `/release-note-none` if
+  not required
+- **do-not-merge/contains-merge-commits:** PR needs to be rebased
+- **needs-rebase:** PR needs to be rebased
+
+PRs that do not have any of the above labeled in this column should be
+evaluated occasionally to see if they are ready for review.
+
+## Waiting on Reviewer
+
+This PR needs review! If you're not sure how to review a PR, start by
+familiarizing yourself with the Kubernetes [pull request guidelines] and
+[review guidelines].
+
+PRs in this column must have the following labels set:
+
+- triage-accepted
+- priority
+- kind
+
+Only [Kubernetes org members] can add an `/lgtm`.
+
+If you want to become an official Node reviewer, you should read through the
+[reviewer responsibilities and requirements].
+
+TODO: Add some node-specific stuff here.
```

Comment:
what do you envision to have here?

Reply:
More node specific content on reviewing, code structure, etc. for components we control would be nice.

Reply:
(Not in the scope of this PR so I'm going to close this)